Factorial_x_n.rb

```diff
@@ -1,6 +1,9 @@
 puts "ingrese mumero"
 n = gets.to_i
-
-n.downto(1) do |i|
-   print "#{i} " if n  % i == 0
+fac = 1
```

Comment:
avoid mixin languages
what is `fac`? , take care of how you name methods and vars

Lets try to use a method instead of having all code in plain,  also if we have the factorial definition we can take advance of `enumerable` module and `array`class in ruby and avoid the `for` loop and get the same result

btw you're not returning the factorial result for n